src/utils/rfq_client.ts

```diff
@@ -31,7 +32,31 @@ const toRfqOrder = (obj: any): RfqOrder => {
     });
 };
 
-export class RfqClient implements IRfqClient {
+// A mapper function to return a serialized OtcOrder into one with BigNumbers
+const toOtcOrder = (obj: any): OtcOrder => {
```

Comment:
can `obj` be something other than `any`?

Reply:
it comes across the wire so I don't think so